gulp/docs.js

```diff
@@ -89,11 +88,60 @@ const componentToDocgen = (
 	},
 });
 
-module.exports = {
-	build: function(callback) {
-		webpack(webpackConfig, callback);
-	},
+function uploadDocsBuild(callback) {
+	// Figure out of the last commit was a tagged version. This command only
+	// succeeds if the last commit has an annotated tag and the tag is output
+	// on stdout if one is found.
+	exec('git describe --abbrev=0 --candidates=0 --tags', function(
+		err,
+		stdoutTag
+	) {
+		var tag = stdoutTag.replace(/\n/g, '');
```

Comment:
`stdoutTag.trim()` maybe?

Reply:
i'll try it...it was from the existing tag parsing logic that i didn't touch, but did move some code around.